factory/ideas.php

```diff
@@ -782,4 +800,24 @@ protected function delete_idea_data($id, $table)
 
 		return (bool) $this->db->sql_affectedrows();
 	}
+
+	/**
+	 * Get the stored idea count
+	 *
+	 * @return int Count of ideas
+ 	 */
+	public function get_idea_count()
+	{
+		return $this->idea_count;
+	}
+
+	/**
+	 * Store an idea count
+	 *
+	 * @param int $idea_count Count of ideas
+	 */
+	public function set_idea_count($idea_count)
```

Comment:
This is only called within the class and is kinda un-needed considering we don't follow proper OOP anyway. Just set the property directly.